challenges/classes.js

```diff
@@ -28,3 +28,22 @@ const cuboid = new CuboidMaker2(4, 5, 5);
 // console.log(cuboid.surfaceArea()); // 130
 
 // Stretch Task: Extend the base class CuboidMaker with a sub class called CubeMaker.  Find out the formulas for volume and surface area for cubes and create those methods using the dimension properties from CuboidMaker.  Test your work by logging out your volume and surface area.
+
+class CubeMaker extends CuboidMaker2 {
+    constructor(size) {
+        super(size, size, size);
+        this.size = size;
+    }
+
+    volume() {
+        return this.size ** 3;
+    }
+
+    surfaceArea() {
+        return this.size * 12;
+    }
+}
+
+const cube = new CubeMaker(5);
+// console.log(cube.volume());
```

Comment:
Amazing job on the sprint challenge Jakub. You have succesfully completed the MVP and even tackled a stretch goal to go beyond the requirements.
All of Your solutions to the assignment look great. Your code is clean and concise. Also nice job using ES6 syntax and arrow functions.
You git workflow is fluid. You made frequent commits with descriptive commit messages.
You are really improving everyday, keep up the good work 👍
Overview:
* Good answers
* Classes challenge completed
* Prototypes challenge completed
* Object-Arrays challenge completed
* Functions challenge completed
* Stretch goal completed
* Fluid git workflow